website/docs/core-concepts/policies.md

```diff
@@ -4,4 +4,23 @@ sidebar_position: 9
 
 # Policies
 
-## Coming Soon
+Policies are mandatory for Dega to work as expected. Default Policies can be added by clicking on Create Default Policies.
```

Comment:
Add section like on the rest of the entities about who/how can one access/edit policy. Meaning, policy to manage policy.